code/datums/perks/job.dm

```diff
@@ -241,12 +255,12 @@
 
 /datum/perk/junkborn
 	name = "Expert Scavenger"
-	desc = "They say there are diamonds in the rough and you can prove it. Removing a trash pile has a chance of revealing a valuable item nobody else would find."
+	desc = "One man's trash is another man's salary. Removing a trash pile has a chance of revealing a valuable item nobody else would find."
 	//icon_state = "treasure" // https://game-icons.net
 
 /datum/perk/ass_of_concrete
-	name = "Expert Defensive Training"
-	desc = "Your position requires you to be unstoppable. Slowing down to a walking pace lets you center your gravity and few things can knock you down."
+	name = "Immovable Object"
+	desc = "Your intense training has perfected your footing, and you're an expert at holding the line. Few things can knock you off balance or push you around, as long as you remember how to anchor yourself to the floor."
```

Comment:
This does not explain how the perk works anymore, as it functions by walking, prior desc said "Slowing down to a walking pace lets" which is a direct explanation on how it works.